quickstarts/post-configure-example/README.md

```diff
@@ -0,0 +1,194 @@
+# JBoss CLI PostConfigure example
+
+This example will explain how to create additional JBoss EAP users using the Post Configure feature available on the
+Container images based on JBoss EAP.
+
+The steps described on this quickstart will work on the following container images:
+
+- RHPAM KIE Server
+- RHDM KIE Server
+- Business Central
+- Business Central Monitoring
+- Decision Central
+- Dashbuilder
+- RHPAM/DM Controller
+
+Note that, this procedure can be applied to any kid of post configure actions.
```

Comment:
Note: This procedure can be applied to any kind of post configure actions.

Reply:
I would also add here: "Complete this procedure after subscribing to the operator and before deploying any KieApp instances"